tutor/commands/compose.py

```diff
@@ -295,77 +297,21 @@ def restart(context: BaseComposeContext, services: t.List[str]) -> None:
     context.job_runner(config).docker_compose(*command)
 
 
+@click.group("do")
+def do() -> None:
+    """
+    Run custom jobs in the right containers.
+    """
```

Comment:
I think supporting `--mount` would make `do` much more powerful. For example, if we incorporated quickdev, it'd be great to be able to write something like:
```
tutor dev do --mount=../xblock-drag-and-drop-v2 pip-install-mounts
```
And if we didn't incorporate quickdev, then we'll still want some sort of task to job to make [setting up a dev env easier](https://docs.tutor.overhang.io/dev.html#setting-up-a-development-environment-for-edx-platform), like:
```
tutor dev do --mount=../edx-platform edx-platform-setup
```

Reply:
Right. It took me a very long time to figure out how to properly call `do` subcommands from quickstart, which is why I decided to get rid of the `--mount` option. But it's actually pretty simple to add it back, so I'll do that now.